transports/upgrades/asq-8to9.md

```diff
@@ -1,16 +1,33 @@
 ---
 title: Azure Storage Queues Transport Upgrade Version 8 to 9
 summary: Migration instructions on how to upgrade Azure Storage Queues Transport from Version 8 to 9.
-reviewed: 2029-11-09
+reviewed: 2020-11-12
 component: ASQ
 related:
 - transports/azure-storage-queues
 - nservicebus/upgrades/7to8
 isUpgradeGuide: true
 upgradeGuideCoreVersions:
- - 8
+ - 7
 ---
 
+## Package name
```

Comment:
Should also have a section about net472 see https://github.com/Particular/docs.particular.net/pull/5114

Reply:
In addition to that it should probably also mention the name space using change similar to

 https://github.com/Particular/docs.particular.net/blob/master/transports/upgrades/sqlserver-5to6.md#moved-types-from-namespace-nservicebustransportsqlserver-to-nservicebustransportsqlserver